test/unit/player.test.js

```diff
@@ -835,6 +835,39 @@ test('createModal()', function() {
   strictEqual(player.children().indexOf(modal), -1, 'modal was removed from player\'s children');
 });
 
+test('should return correct remote text track values', function () {
```

Comment:
Not sure where this should go or how this should be tested.

- Note: `techOrder: ['html5']`

---

`techFaker` doesn't initialize the `<track>` correctly, but fires the `player.addRemoteTextTrack` correctly. Open to recommendations.